x-pack/plugins/security_solution/cypress/e2e/explore/dashboards/entity_analytics.cy.ts

```diff
@@ -304,46 +303,42 @@ describe('Entity Analytics Dashboard', () => {
     });
   });
 
-  // tracked by https://github.com/elastic/kibana/issues/161874
-  describe.skip('With anomalies data', () => {
+  describe('With anomalies data', () => {
     before(() => {
       cy.task('esArchiverLoad', 'network');
+      cy.intercept('POST', 'internal/ml/results/anomaly_search').as('anomalies1');
```

Comment:
I've no idea why these intercepts are displayed here, but I removed them and are not present on the code, it would be nice if someone can checkout the branch to make sure that are not present.

Reply:
I checked the branch and I see the intercepts in the codebase. Git blame says it was introduced here: https://github.com/MadameSheema/kibana/commit/b633a9887f57b3e5ff4e39c447ef861d47863f9b#diff-ebbf1fe43a192fa8847b68d54b536846aab97118986c932308f6e594fdd9cb86R310
<img width="1575" alt="Screenshot 2023-07-26 at 11 28 36" src="https://github.com/elastic/kibana/assets/1490444/5a09183f-70ee-49cd-8f07-9e132549c82f">